.circleci/config.yml

```diff
@@ -964,6 +964,11 @@ workflows:
           suite: itest-paych_cli
           target: "./itests/paych_cli_test.go"
       
+      - test:
+          name: test-itest-pending_deal_allocation_retrieval
```

Comment:
Let's not have `retrieval` as a substring of the file name, that has a certain implication for us.